decidim-admin/app/controllers/concerns/decidim/admin/content_blocks/landing_page.rb

```diff
@@ -3,6 +3,11 @@
 module Decidim
   module Admin
     module ContentBlocks
+      # i18n-tasks-use t('decidim.admin.organization_homepage.edit.add')
+      # i18n-tasks-use t('decidim.admin.organization_homepage.edit.destroy_confirmation')
+      # i18n-tasks-use t('decidim.admin.organization_homepage.edit.active_content_blocks')
+      # i18n-tasks-use t('decidim.admin.organization_homepage.edit.inactive_content_blocks')
+      # i18n-tasks-use t('decidim.admin.organization_homepage.edit.title')
```

Comment:
I prefer not to introduce these `i18n-tasks-use` if we can actually avoid it. In this case I'd live with a bit of duplication in these definitions. 

These `i18n-tasks-use` ignores add mini-overheads down in the road: if someone changes a method 65 lines below then they need to remember to actually change this, and also it's easy to miss by reviewers as they're out of the view of the diff. 

I prefer to:

1. remove them altogether, and inline the `i18n_scope` method 
2. in this case, I understand that's not possible/easy as the intention of the `i18n_scope` method is to actually be overridden by a class that's using this mixin, at least according to the comment. In this case I'd prefer to add the `i18n-tasks-use` definition in that method, as these keys are used in just that method, and not used in multiple places of the class (as long as I see/understand)